Octokit/Clients/RepositoryCommitsClient.cs

```diff
@@ -69,15 +69,40 @@ public Task<IReadOnlyList<GitHubCommit>> GetAll(string owner, string name)
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="request">Used to filter list of commits returned</param>
+        /// <param name="options">Options for changing the API response</param>
+        /// <returns></returns>
+        public Task<IReadOnlyList<GitHubCommit>> GetAll(string owner, string name, ApiOptions options)
+        {
+            return GetAll(owner, name, new CommitRequest(), options);
+        }
+        
+        /// <summary>
+        /// Gets all commits for a given repository
+        /// </summary>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="request">Used to filter list of commits returned</param>
         /// <returns></returns>
         public Task<IReadOnlyList<GitHubCommit>> GetAll(string owner, string name, CommitRequest request)
         {
+            return GetAll(owner, name, request, ApiOptions.None);
```

Comment:
Could you add here:
```cs
Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
Ensure.ArgumentNotNullOrEmptyString(name, "name");
```

We have discussion about it [here](https://github.com/octokit/octokit.net/pull/1218/files#r57460248).